### PR TITLE
[10.0][MIGRATION] Migrating l10n_ch_states and l10n_ch_zip

### PR DESCRIPTION
**Pull request by BT-aestebanez**

### CODE REVIEW

**Comment by leemannd** [comment]
LGTM, but should we deactivate test on DTA while it's unported? cc @yvaucher

File: l10n_ch_states/__manifest__.py [hunk 1]
```diff
@@ -22,15 +22,19 @@
     'name': 'Switzerland Country States',
     'category': 'Localisation',
     'summary': '',
-    'version': '9.0.1.0.0',
-    'author': 'copado MEDIA UG, Odoo Community Association (OCA)',
+    'version': '10.0.1.0.0',
+    'author': '''
+        copado MEDIA UG,
+        brain-tec AG,
```
Comment (yvaucher) [inline l10n_ch_states/__manifest__.py:28]: I don't see a significant enough addition to the module to set brain-tec as author here. Please remove this line.
While you contribute, you are not an author of the module.

**Comment by yvaucher** [comment]
This PR is redundant with #287

File: l10n_ch_states/__manifest__.py (hunk 1 above)
Comment (BT-aestebanez) [reply l10n_ch_states/__manifest__.py:28]: Yes, you are right

**Comment by BT-aestebanez** [comment]
@yvaucher Ok, so I think the best idea would be apply these small changes in the PR 287 and close this one, do you agree?

**Comment by yvaucher** [comment]
I see that there are a few missing changes both in this PR and in #287, I'll do the portage for both modules. I want to clean few things like copyright headers and readme.
Plus I'll make a PR per module.

**Comment by BT-aestebanez** [comment]
Ok, thanks

**Comment by yvaucher** [comment]
superseeded by #292 and #293

**Pull request closed without merging**